client/src/app/pages/questionnaires/Questionnaires.tsx

```diff
@@ -0,0 +1,480 @@
+      required: false,
+      system: true,
+    },
+    {
+      id: 2,
+      name: "Custom questionnaire",
+      questions: 24,
+      rating: "15% Red, 35% Yellow",
+      dateImported: "9 Aug. 2023, 03:32 PM EST",
+      required: true,
+      system: false,
+    },
+    {
+      id: 3,
+      name: "Ruby questionnaire",
+      questions: 34,
+      rating: "7% Red, 25% Yellow",
+      dateImported: "10 Aug. 2023, 11:23 PM EST",
+      required: true,
+      system: false,
+    },
+  ]);
+  const { questionnaires, isFetching, fetchError } =
+    useFetchQuestionnaires(mockQuestionnaires);
+
+  const onSaveQuestionnaireSuccess = () => {};
+
+  const onSaveQuestionnaireError = (error: AxiosError) => {
+    pushNotification({
+      title: getAxiosErrorMessage(error),
+      variant: "danger",
+    });
+  };
+
+  const { mutationFn: updateQuestionnaire } = useUpdateQuestionnaireMutation(
+    onSaveQuestionnaireSuccess,
+    onSaveQuestionnaireError
+  );
+
+  const onDeleteQuestionnaireSuccess = (name: string) => {
+    pushNotification({
+      title: t("toastr.success.deletedWhat", {
+        what: name,
+        type: t("terms.questionnaire"),
+      }),
+      variant: "success",
+    });
+  };
+
+  const { mutationFn: deleteQuestionnaire } = useDeleteQuestionnaireMutation(
+    onDeleteQuestionnaireSuccess,
+    onSaveQuestionnaireError
+  );
+
+  const [isImportModal, setIsImportModal] = React.useState<boolean>(false);
+  const [isDownloadTemplateModal, setIsDownloadTemplateModal] =
+    React.useState<boolean>(false);
+
+  const [isKebabOpen, setIsKebabOpen] = React.useState<number | null>(null);
+  const [questionnaireToExport, setQuestionnaireToExport] = React.useState<
+    number | null
+  >(null);
+  const [questionnaireToDelete, setQuestionnaireToDelete] = React.useState<
+    number | null
+  >(null);
+
+  const tableControls = useLocalTableControls({
+    idProperty: "id",
+    items: questionnaires,
+    columnNames: {
+      required: "Required",
+      name: "Name",
+      questions: "Questions",
+      rating: "Rating",
+      dateImported: "Date imported",
+    },
+    isSelectable: false,
+    expandableVariant: null,
+    hasActionsColumn: true,
+    filterCategories: [
+      {
+        key: "name",
+        title: t("terms.name"),
+        type: FilterType.search,
+        placeholderText:
+          t("actions.filterBy", {
+            what: t("terms.name").toLowerCase(),
+          }) + "...",
+        getItemValue: (item) => {
+          return item?.name || "";
+        },
+      },
+    ],
+    sortableColumns: ["name", "dateImported"],
+    getSortValues: (assessment) => ({
+      name: assessment.name || "",
+      dateImported: assessment.dateImported || "",
+    }),
+    initialSort: { columnKey: "name", direction: "asc" },
+    hasPagination: true,
+    isLoading: isFetching,
+  });
+  const {
+    currentPageItems,
+    numRenderedColumns,
+    selectionState: { selectedItems },
```

Comment:
You can remove this line that pulls out `selectedItems` since we're not using selection state on this page.